QUEST.tsv

```diff
@@ -122,82 +122,82 @@ xQUEST_20150317_000121	This mission makes me feel like I've seen enough monsters
 xQUEST_20150317_000122	Chupacabras are fond of lush vegetation and upper streams.
 xQUEST_20150317_000123	Klaipeda Guard Captain
 xQUEST_20150317_000124	You're the one who…
-xQUEST_20150317_000125	It looks like you can see into the upper forest from here. {nl}Try not to provoke anything bigger than you.
-xQUEST_20150317_000126	You don't have the required quest item. Are you sure you want to continue?
+xQUEST_20150317_000125	It looks like you can see into the upper forest from here. {nl}Try not to get into trouble.
```

Comment:
The edited version is fine here. The previous translation was pulled out of no where, the Korean doesn't mention that.